src/components/form/widgets/date-picker.ts

```diff
@@ -61,7 +61,7 @@ export class DatePicker extends React.Component<DateWidgetProps> {
         const formatMapping = {
             'date-time': 'YYYY-MM-DDTHH:mm:ssZ',
             date: 'YYYY-MM-DD',
-            time: 'HH:mm:ss',
+            time: 'HH:mm:ssZ',
```

Comment:
The meaning of time zone offset when converting to local time is ambiguous without knowing the date, so I don't think it makes sense to include the time zone for the time picker. Unless we made it possible to also provide the date somehow 🤔

Reply:
da212df8b1a655fe65fe50395edfc8f4b32f776c